src/file_change_handler.ts

```diff
@@ -71,4 +80,12 @@ export class FileChangeHandler {
 			id: "" // TODO: Fix this, should be optional!
 		}
 	}
+
+	private isDartLike(document: vscode.TextDocument): boolean {
```

Comment:
[Done](https://github.com/DanTup/Dart-Code/commit/f81cb14a0844d4f69966cba728d88a67001b6a6e) this using the langageID and added `basename()` around the filename since it was getting the full path.